packages/types/src/config/PowerSyncConfig.ts

```diff
@@ -207,6 +222,16 @@ export const powerSyncConfig = t.object({
     })
     .optional(),
 
+  service: t
+    .object({
+      health_checks: t
+        .object({
+          probe_modes: t.array(t.Enum(ProbeType)).optional()
```

Comment:
After some consideration. I think boolean flags here might be more convenient. This would allow devs to use a base YAML config which conditionally configures the probe modes with environment variables.

```yaml
service:
    health_checks:
       probes:
          http: true
          filesystem: !env PS_ENABLE_FILESYSTEM_PROBES

```

Reply:
I've updated this to use booleans